src/backend/utils/graph_generation.c

```diff
@@ -195,23 +193,22 @@ Datum create_complete_graph(PG_FUNCTION_ARGS)
     vtx_seq_id = get_relname_relid(vtx_seq_name_str, nsp_id);
     edge_seq_id = get_relname_relid(edge_seq_name_str, nsp_id);
 
-    props = create_empty_agtype();
```

Comment:
This shouldn't be removed. A previous PR moved this here, from inside of the loop.